### PR TITLE
[1LP][RFR]Improving test coverage

### PR DESCRIPTION
**Pull request by anurag03**
Purpose or Intent
=================
Here the test case is dynamically parametrised to improve the test coverage.

{{pytest: -v -k 'test_infra_grid_page_per_item or test_infra_tile_page_per_item or test_infra_list_page_per_item or test_infra_report_page_per_item'}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 474ba77e70c5d9a68828b5a1bb021fb05ff45ed2:

`cfme/tests/configure/test_visual_infra.py`:
- :bangbang: Line 37:1: **E305** *expected 2 blank lines after class or function definition, found 1*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4696bcfa357db8a94f9addb88ecd1572b2bb7fc0:

`cfme/tests/configure/test_visual_infra.py`:
- :bangbang: Line 37:1: **E305** *expected 2 blank lines after class or function definition, found 1*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by anurag03** [comment]
The test case works fine for 5.7 when tested locally test case related to report is passing. Hence changing the state from [WIPTEST] to [RFR].

File: cfme/tests/configure/test_visual_infra.py [hunk 1]
```diff
@@ -2,11 +2,10 @@
 
 from copy import copy
 import pytest
-
+import itertools
```
Comment (quarckster) [inline cfme/tests/configure/test_visual_infra.py]: **REQUIRED**
Please sort imports alphabetically

File: cfme/configure/settings.py [hunk 2]
```diff
@@ -269,6 +269,34 @@ def timezone(self, value):
         if view.visualdisplay.time_zone.fill(value):
             view.save.click()
 
+    @property
+    def grid_view_read(self):
```
Comment (quarckster) [inline cfme/configure/settings.py]: **REQUIRED**
Bad naming is here. You named the property like a method. It should be something like `grid_view_entities`.

**Pull request merged**